sdks/builds/ios.mk

```diff
@@ -11,19 +11,19 @@
 
 PLATFORM_BIN=$(XCODE_DIR)/Toolchains/XcodeDefault.xctoolchain/usr/bin
 
-_ios_CFLAGS= \
+ios_CFLAGS= \
```

Comment:
These start with an underscore because they are used internally inside the template macros.

Reply:
They are not inside the template, that's why I removed the `_`, but feel free to revert it back, I don't have a strong opinion about it